WebRoot/css/zm.css

```diff
@@ -7281,30 +7281,30 @@ input[type="radio"] + label {
 	@WhiteBlackListViewItemsNumberUsed@
 }
 
-.ZOptionsSectionMain .ZmBlockMessage {
+.ZOptionsSectionMain .ZmPrefMailBlockMessage {
 	@PrefMailBlockMessageCell@
 }
 
-.ZOptionsLabelTop.ZmComposeLabel {
+.ZOptionsLabelTop.ZmPrefMailComposeLabel {
 	@PrefMailComposeLabel@
 }
 
-.ZmPrefMailCompose .ZOptionsLabelNarrow {
+.ZmPrefMailComposeContent .ZOptionsLabelNarrow {
 	@PrefMailComposeRowLabel@
 }
 
-.ZmPrefMailCompose .ZOptionsField {
+.ZmPrefMailComposeContent .ZOptionsField {
 	@PrefMailComposeRowValueWrapper@
 }
 
-.ZmPrefMailCompose .ZOptionsField .ZSelect > table {
+.ZmPrefMailComposeContent .ZOptionsField .ZSelect > table {
 	@PrefMailComposeRowSelectWidget@
 }
 
-.ZmPrefMailCompose .ZWidgetTable {
+.ZmPrefMailComposeContent .ZWidgetTable {
```

Comment:
Changed the classname to ZmPrefMailComposeColorPicker .
https://github.com/Zimbra/zm-web-client/pull/102/commits/58bc858a52d6225549351a9fb2dbb96ad0f1ac08